### PR TITLE
Update dependencies

### LINKED ISSUE #47: Solve conflict with dependencies regarding Bootstrap4

### PR DESCRIPTION
**Pull request by Gal-f**
Updated Pipfile and Pipfile.lock to include Bootstrap4 and resolve conflict that existed
and Pytest.

fix #47

### CODE REVIEW

**Comment by Gal-f** [comment]
The Flake8 complaint about unknown class Category was fixed in PR #44 ,
should I commit amend a fix in this PR as well, or would that just create a conflict down the road? (for changing the same lines in different PRs) @ravidbro @ifireball @Omeramsc

**Comment by ravidbro** [comment]
If this PR is about 'update dependencies' then the messages model commit should not be in this branch.
you should have only one commit to fix dependencies that have only 'main' and this change.
You should create a new branch from 'main' (checkout -b) and fix only that.

**Comment by ravidbro** [comment]
now you have a conflict you shouldn't have if the non-relevant commit weren't here in the branch

**Comment by Gal-f** [comment]
> now you have a conflict you shouldn't have if the non-relevant commit weren't here in the branch



> If this PR is about 'update dependencies' then the messages model commit should not be in this branch.
> you should have only one commit to fix dependencies that have only 'main' and this change.
> You should create a new branch from 'main' (checkout -b) and fix only that.

That's right. The second commit shouldn't have been there in the first place.
That's the reason to my question on Slack about removing a commit.
I wasn't successful in removing it so far.

**Comment by Gal-f** [comment]
I think I've manged to kill the notorious commit. 
Please make sure I didn't make any damage before merging... 🙏  @ravidbro

**Review by Omeramsc — APPROVED** [review]
LGTM
I don't see the unrelevant commit.

**Comment by ravidbro** [comment]
I'm so sorry for another comment on this small PR, but the commit title is 'adding bootstrap' and when I look in the content I see that you are adding only 'pytest'

File: Pipfile.lock [hunk 1]
```diff
@@ -23,13 +23,29 @@
...
-                "sha256:14a4b7cd77297fba516fc0d92444cc2e2e388aa9de32d7a68d4a83d58f5a4927",
-                "sha256:14b87775ffedab2ef6299b73343d1b4b41e5d4e2aa58c6581f114dbec01e3f8f"
+                "sha256:5c866205f15e7a7123f1eec6ab939d22d5bde1416635cab259684af66d8e48a2",
+                "sha256:edb10b5c45e7e9c0fb1dc00b76ec7449aca258a39ffd613dbd078c51d19c9f03"
             ],
             "index": "pypi",
-            "version": "==3.1.3"
+            "version": "==3.1.4"
+        },
+        "django-bootstrap4": {
```
Comment (Gal-f) [inline Pipfile.lock:42]: @ravidbro Bootstrap4 is added here.
The original PR by @or313 had Bootstrap in pipfile but not in pipfile.lock.
This commit merely fixes this minor mistake.
Pytest is added as well.

**Pull request merged**